src/components/connected/Welcome.js

```diff
@@ -25,201 +25,151 @@ function Welcome() {
   return (
     <>
       <div className="welcome">
-        <div className="fr-container fr-container--fluid fond-sombre">
-          <div className="fr-grid-row">
-            <div className="fr-col-12">
-              <div className="fr-container">
-                <div className="fr-grid-row fr-grid-row--center">
-                  <div className="fr-col-12 fr-mt-15w fr-mb-3w">
-                    { conseiller && !conseiller?.supHierarchique &&
-                      <div className="info-rappel-hierarchique fr-mb-3w fr-py-1w">
-                        <i className="ri-information-line information icone-info-rappel"></i>
-                        <p className="margin-info">
-                          Afin que votre responsable hi&eacute;rarchique puisse recevoir toutes les informations concernant le dispositif et vous accompagner
-                          au mieux, merci de nous indiquer ses coordonn&eacute;es en compl&eacute;tant le formulaire dans l&apos;onglet Mes informations,
-                          contact hi&eacute;rarchique sur la page d&apos;accueil en haut &agrave; droite de votre Espace Coop.
-                        </p>
-                      </div>
-                    }
-                    <h2 className="titre">
-                      <img src="/logos/home-connected/hexagone-1.svg" className="hexagone"/>
-                      Services
-                    </h2>
-                  </div>
-                </div>
+        { conseiller && !conseiller?.supHierarchique &&
```

Comment:
Normalement ca devrait suffire : 
```suggestion
        { !conseiller?.supHierarchique &&
```